src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -224,5 +224,11 @@ private void RaiseEvents(Project project, ImmutableArray<DiagnosticData> diagnos
                         ValueTuple.Create(this, kv.Key), workspace, solution, project.Id, kv.Key, kv.ToImmutableArrayOrEmpty()));
             }
         }
+
+        public override bool ContainsDiagnostics(Workspace workspace, ProjectId projectId)
+        {
+            // for now, it always return false;
```

Comment:
At some point we need to either delete EngineV2 OR reuse it for your out-of-proc work.